test/e2e/commands/node_local_hedera.test.ts

```diff
@@ -42,9 +43,26 @@ argv[flags.localBuildPath.name] = 'node1=../hedera-services/hedera-node/data/,..
 argv[flags.namespace.name] = LOCAL_HEDERA
 
 e2eTestSuite(LOCAL_HEDERA, argv, undefined, undefined, undefined, undefined, undefined, undefined, true, (bootstrapResp) => {
+  const nodeCmd = bootstrapResp.cmd.nodeCmd
+  const accountCmd = bootstrapResp.cmd.accountCmd
   describe('Node for hedera app should have started successfully', () => {
     hederaK8 = bootstrapResp.opts.k8
 
+    it('save the state and restart the node with saved state', async function () {
```

Comment:
are we able to validate that after the state is restored that the new accounts still exist?